src/index.ts

```diff
@@ -287,6 +287,11 @@ export async function latencies(
   await sql`SELECT ${now} AS timenow_uncast`; // us: '2023-05-26T13:35:22.616Z', pg: '2023-05-26T14:35:22.616+01:00' <-- different representations
   await sql`SELECT ${now}::timestamp AS timestampnow`; // us: 2023-05-26T12:35:22.696Z, pg: 2023-05-26T13:35:22.696Z <-- different TZs
 
+  // composed sql for literal strings (SQL injection risk)
+  const func = 'pi()';
+  await sql`SELECT ${sql(func)} AS composed_func_result`;
+  await sql`SELECT ${11} AS eleven, ${sql(func)} AS composed_func_result, ${12} AS twelve`;
```

Comment:
Would it make sense to have a test case with a `try...  catch...`  using a "Tagged-template query with parameters" to make sure we are handling that right?